modules/use-cases-architectures/pages/real-time-data-pipeline/03-put-it-all-together.adoc

```diff
@@ -138,23 +133,19 @@ Similar to how you followed the above flow of raw click data, follow this flow t
 . Navigate to your {product} tenant and check the production/product-clicks topic.
 .. There should be 1 message in “Data In” and 1 message in “Data Out”.
 
-. Finally, navigate to your {astra-db} database's CQL Console.
-.. Query the product_clicks table:
+. Inspect the data in your {astra-db} database.
 +
-[tabs]
-====
-CQL::
+In the {astra-ui}, go to your `webstore-clicks` database, click **CQL Console**, and then query the `product_clicks` table:
 +
 --
 [source,sql,subs="attributes+"]
 ----
 select * from click_data.product_clicks;
 ----
---
-
-Result::
 +

```

Comment:
There are some passthru issues with the text in the section below, but it's out of scope of this PR.
<img width="856" height="747" alt="Screenshot 2025-07-15 at 1 30 52 PM" src="https://github.com/user-attachments/assets/9fd3212f-9c73-44d3-b295-3b93c03aabea" />